openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/fabric8/F8OpenShiftAdapter.java

```diff
@@ -112,7 +112,7 @@ static OpenShiftConfig toOpenShiftConfig(CubeOpenShiftConfiguration configuratio
             .withMasterUrl(configuration.getKubernetesMaster())
             .withTrustCerts(configuration.isTrustCerts());
 
-        if (configuration.hasOpenshiftBasicAuth()) {
+        if (configuration.hasBasicAuth()) {
```

Comment:
Same for this class, my PR comes with the final solution by reusing the instance.